lib/agent.js

```diff
@@ -324,6 +325,23 @@ Agent.prototype.start = function (opts) {
 
   this.logger.info(preambleData, 'Elastic APM Node.js Agent v%s', version);
 
+  if (!logging.isLoggerCustom(this.logger)) {
+    // Periodically dump the current config (delta from defaults) when logging
+    // at "trace"-level. This allows getting the effective config from a running
+    // agent by setting trace-level logging and getting 1 minute of logs.
+    // (Sometimes getting logs from application *start* is no possible.)
+    setInterval(() => {
+      if (this.logger.isLevelEnabled('trace')) {
```

Comment:
could the level be changed at runtime? If not I think we could `setInterval` within this block.

Reply:
Yes it can be changed at runtime via central config, and in the use case I needed this for that *was* the case.
I balked at integrating this change with central config handling. 

However, I may do that for the *main* branch version of this change, because having a `setInterval` running all the time kinda sucks. On the other hand, having a setInterval (change in behaviour, however slight) that only turns on in "trace" made could mean more subtle bugs. It would be awful if there was a crash that only resulted from code that enabled when in "trace" mode. I'm talking generally.